src/common/functions/format.ts

```diff
@@ -0,0 +1,7 @@
+export const formatPercentage = (amount: number) => {

```

Comment:
Do you think we should keep one function to one file? I think having a file for each makes it easier to glance what are the available functions. Wdyt?

Reply:
sounds good, updated at 7aca4ea and added this to the functions readme at d4aaec1